api/src/test/java/io/opentelemetry/common/LabelsTest.java

```diff
@@ -124,4 +124,14 @@ public void builder() {
                 "key1", "value1",
                 "key2", "value2"));
   }
+
+  @Test
+  public void toBuilder() {
+    Labels initial = Labels.of("one", "a");
+    Labels second = initial.toBuilder().setLabel("two", "b").build();
+    assertThat(initial.size()).isEqualTo(1);
```

Comment:
Looks like we don't really need the size assertions

Reply:
Well maybe they help - don't think I can find a `toString` for the key value pair classes which would be important for the equals assertions when they fail.

Reply:
My thinking was that, with such a non-trivial implementation internally, that a few extra assertions wouldn't hurt. The size isn't the size of the arraylist, for example. 

Also, a toString is a really good idea! Not sure why I didn't put that in originally.

Reply:
I added a toString, just for good measure.